docs/swupd.1.rst

```diff
@@ -222,6 +222,93 @@ SUBCOMMANDS
         Optionally set the top-level directory for the swupd-managed system.
         If not specified this will default to '/'
 
+``os-install``
+
+    Perform system software installation in the specified location. Install
+    all files into {path} as specified by the `swupd os-install {path}` option.
+    Useful to generate a new system root.
+
+    - `-V, --version`
+
+        Install OS version V.
+
+    - `-x, --force`
+
+        Attempt to proceed even if non-critical errors found.
+
+    - `-B, --bundles=[BUNDLES]`
+
+        Include the (comma separated) list of bundles with the base OS install.
+
+        Examples:
+
+        - ``--bundles xterm,vi``
+
+            Installs bundles os-core and vi, along with os-core (installed by default).
+
+``repair``
+
+    Correct any issues found. This will overwrite incorrect file content,
+    add missing files and do additional corrections, permissions, etc.
+
+    - `-m, --manifest`
```

Comment:
This sounds like the same meaning than `-V, --version` as used by `os-install`. If true, it would be better (i.e. less error-prone and confusing) to have both `os-install` and `repair` use the same.

Reply:
yes. you're write. we should have used -m in os-install

Reply:
@otaviobp I changed `-m` in `os-install` to be `-V`/`--version` because IMO it makes more sense. When you are installing an OS you are really thinking about choosing a version of it, not really a manifest of it. Maybe it made more sense for the `verify` command when you were verifying against one specific manifest. I think referencing "manifests" in those commands may cause confusion to users not familiar with swupd & mixer and therefore manifests. Maybe it would be better to also change the -m/--manifest option from the `repair` and `diagnose` commands to also be `-V`.`--version`. This will be a somewhat more disruptive change for users that are used to use `-m`, but since we are doing a disruptive change here I guess now is a good time to do it.